Purchases/Attribution/RCAttributionTypeFactory.m

```diff
@@ -29,35 +30,35 @@ - (NSString *)mangledTrackingClassName {
     return @"NGGenpxvatZnantre";
 }
 
-- (Class<FakeTrackingManager> _Nullable)atTrackingClass {
+- (Class<FakeFollowingManager> _Nullable)atFollowingClass {
     // We need to do this mangling to avoid Kid apps being rejected for getting idfa.
     // It looks like during the app review process Apple does some string matching looking for
     // functions in ATTrackingTransparency. We apply rot13 on these functions and classes names
     // so that Apple can't find them during the review, but we can still access them on runtime.
-    NSString *className = [self rot13:self.mangledTrackingClassName];
+    NSString *className = [self.class rot13:self.mangledTrackingClassName];
 
-    return (Class<FakeTrackingManager> _Nullable)NSClassFromString(className);
+    return (Class<FakeFollowingManager> _Nullable)NSClassFromString(className);
 }
 
-- (Class<FakeASIdentifierManager> _Nullable)asIdentifierClass {
+- (Class<FakeASIdManager> _Nullable)asIdClass {
     // We need to do this mangling to avoid Kid apps being rejected for getting idfa.
     // It looks like during the app review process Apple does some string matching looking for
     // functions in the AdSupport.framework. We apply rot13 on these functions and classes names
```

Comment:
maybe we should remove AdSupport.framework and idfa in the comment? or are we not worried about that?

Reply:
can't hurt

Reply:
Last time I checked, comments don't get compiled 😄

Reply:
right, forgot for a minute that the binary that makes it to app store builds won't have the comments in it